src/modules/python/pbs/v1/_base_types.py

```diff
@@ -533,6 +533,9 @@ def __sub__(self,other):
 	# and not in _size's richcompare.
         return size(s.__sub__(o))
 
+    def __deepcopy__(self, mem):
+        return size(str(self))
```

Comment:
I don't see the str() method defined for this type.  What will it return?

Reply:
It returns a string representing the size. It's implemented in pbs_python_svr_size_type.c.